### ISSUE TITLE
BETTER -- make report URL available to anyone who it is sent to.

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#3321]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

<!--Add any other context or screenshots about the feature request here.-

@areagan-cob

**Comment by RDmitchell** [comment]
From @areagan-cob 

Ideally there would be a field in SEED with the "public" URL so that the city could pass that URL field to Salesforce to be included in their automated emails to building owners.

**Comment by nllong** [comment]
Agreed that a public URL would be nice, but that really isn't an option at the moment. You can download the HTML file from the 'Full Analysis' details of any run. See the download icon next to "Building Report" below. If it needs to be an HTML, then the user could print the downloaded HTML as PDF. 

<img width="1015" alt="image" src="https://user-images.githubusercontent.com/1907354/175076615-486a03d1-f993-42a7-a13f-496b5b910d6d.png">

**Comment by RDmitchell** [comment]
@nllong -- is this something we could develop in the next few months? 

Alternatively, if that isn't possible, could we have SEED generate the PDF as on of the download options?

**Comment by nllong** [comment]
Having SEED convert the HTML to PDF and download would be easier (and safer).

**Comment by RDmitchell** [comment]
Seems like that might be a good first step

**Comment by RDmitchell** [comment]
Then the question is whether that PDF file (or a reference to it) could be stored in a SEED field, which is what Berkeley needs so that it works with their automated Salesforce system
